app/assets/stylesheets/media.scss

```diff
@@ -112,42 +112,6 @@
   max-height: calc(100vh - 100px); /* Scroll height for metadata when the auth wrapper is displaying */
 }
 
-.authLinkWrapper {
-  align-items: center;
-  display: flex;
-  background-color: var(--active-color);
-  color: white;
-  padding: .5rem;
-
-  .loginMessage {
-    margin-bottom: 0;
-    margin-left: 1rem;
-    margin-top: 0;
-  }
-
-  button {
-    margin-left: 0.75rem;
-    background-color: white;
-    color: var(--active-color);
-    border: 0;
-    border-radius: 5px;
-    padding: 0.2rem 0.4rem;
-    text-transform: uppercase;
-    &:hover {
-      cursor: pointer;
-      opacity: 0.9;
-    }
-  }
-
-  .dismissButton {
-    padding: 0.2rem;
-    svg {
-      font-size: 1rem;
-      vertical-align: bottom;
-    }
-  }
-}
-
```

Comment:
we need this CSS for the login panel, which is also used by the PDF viewer, so move it to the `companion_window` (as shown above)